internal/cmd/fluxtest-harness-influxdb/test.go

```diff
@@ -0,0 +1,244 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+
+	"github.com/influxdata/flux"
+	"github.com/influxdata/flux/ast"
+	"github.com/influxdata/flux/cmd/flux/cmd"
+	"github.com/influxdata/flux/csv"
+	"github.com/influxdata/flux/execute/table"
+	"github.com/influxdata/flux/parser"
+	fluxClient "github.com/influxdata/influxdb/flux/client"
+	"github.com/influxdata/influxdb/tests"
+	"github.com/spf13/cobra"
+)
+
+type testExecutor struct {
+	ctx         context.Context
+	s           tests.Server
+	writeOptAST *ast.File
+	readOptAST  *ast.File
+	errOutput   bytes.Buffer
+	i           int
+	failed      bool
+}
+
+func NewTestExecutor(ctx context.Context) (cmd.TestExecutor, error) {
+	e := &testExecutor{ctx: ctx}
+	e.init()
+
+	config := tests.NewConfig()
+	config.HTTPD.FluxEnabled = true
+	config.HTTPD.FluxLogEnabled = true
+	config.HTTPD.FluxTestingEnabled = true
+
+	// NOTE: This will panic on failure. Not changing it for now.
+	e.s = tests.OpenServer(config)
```

Comment:
I'm torn between this approach and spinning up a separate `Influxd` process for a real integration test.

Reply:
I'd vote for leaving it as-is for now, to mirror 2.x.